src/oc_chain_reporter.erl

```diff
@@ -0,0 +1,32 @@
+%%%------------------------------------------------------------------------
+%% Copyright 2017, OpenCensus Authors
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%% http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%
+%% @doc This module allows reporters chaining.
+%% @end
+%%%-----------------------------------------------------------------------
+-module(oc_chain_reporter).
+
+-behavior(oc_reporter).
+
+-export([init/1,
+         report/2]).
+
+%%-
```

Comment:
specs would be nice to have specs as the config is well defined (same for report).